packages/instance-model/index.d.ts

```diff
@@ -100,6 +100,7 @@ declare class MongoDBInstance extends MongoDBInstanceProps {
   }): Promise<Collection | null>;
   removeAllListeners(): void;
   on(evt: string, fn: (...args: any) => void);
+  off(evt: string, fn: (...args: any) => void);
```

Comment:
lol :smile: https://github.com/mongodb-js/compass/pull/5102/files#diff-a79e9b0455facd1015dd43ba2327be7c140c049fcbb16ef624e15a67dfc38cbaR103 no strong feelings either way will align that with this PR ;)

Reply:
ah actually, now I remember why I did this; `AppRegistry` has `removeListener` but not `off`. Should we also add `off` to `AppRegistry` then?

Reply:
We can do that for consistency, yes, especially now that we use it so much! With ampersand models you have off, but not removeListener, so this is why I added it here

Reply:
Renamed to align with your PR in https://github.com/mongodb-js/compass/pull/5100/commits/6691588aff5e4ba93031a41042309b9be3b182e0